### DISCUSSION TITLE
[Feature Request] Auto-expand large file diffs if they have comments

### DISCUSSION THREAD

**Discussion by HeatherLemieux** [community/community#65214] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

My team and I often have PRs that result in large file diffs. I understand the need to reduce server load by not rendering large diffs by default, however if those files have code comments they should no longer be collapsed automatically. This behavior is super frustrating and can result in missing important comments. 

The Conversations dropdown is helpful in that it allows you to view all the comments, however, if the comment is part of a file that is collapsed it will not expand the file and navigate to it; it does nothing which was reported as a bug [here](https://github.com/orgs/community/discussions/8531). I also know that you can `alt+click` to expand all files in a PR, however this would also expand all files with generated code and any files that don't have comments which can be ignored (in 95% of cases). I would love if github changed the default to automatically expand all files that have comments, or at least have an option to expand all files with comments via a single click. 

Thanks!

**Comment by rchrdgrace** [comment]
Cannot agree more. This is a basic capability that somehow has never made it into this critical-path product. Maybe AI will fix this! Then Microsoft can fire the coders who didn't do this basic design. Complete lack of user awareness.

**Reply by mryanprice** [reply]
I would also really like to see this. 

It’s super depressing to take the time to open a “ Large diffs are not rendered by default.” , find a bug, comment on it, and know there is no chance anyone will ever see it. 

On my company’s on-premise GitHub alt-click doesn’t expand “ Large diffs are not rendered by default”
